monai/apps/pathology/data/datasets.py

```diff
@@ -32,6 +32,7 @@ class PatchWSIDataset(Dataset):
         region_size: the size of regions to be extracted from the whole slide image.
         grid_shape: the grid shape on which the patches should be extracted.
         patch_size: the size of patches extracted from the region on the grid.
+        level: the level at which the patches to be extracted (default to 0).
```

Comment:
As you changed the arg to `kwargs`, could you please help update the doc-string to `kwargs` here?
And also add doc-string to other datasets?

Thanks.